frontend/src/components/resourceMap/details/KubeNodeDetails.tsx

```diff
@@ -99,6 +109,18 @@ export const KubeObjectDetails = memo(
         ([key]) => key.toLowerCase() === kind?.toLowerCase()
       )?.[1] ?? DetailsNotFound;
 
+    let content = <Component name={name} namespace={namespace} />;
+
+    if (customResourceDefinition) {
```

Comment:
Should we set the content for the non-CRD only on else here? To avoid calling the function and then rewriting it.

Reply:
replaced with a ternary op
but also creating jsx element is pretty much free, it doesn't call the component function, just creates an object